src/OrchardCore.Modules/OrchardCore.Security/Services/PermissionsHeaderPolicyProvider.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Microsoft.AspNetCore.Http;
+
+namespace OrchardCore.Security.Services
+{
+    public class PermissionsHeaderPolicyProvider : HeaderPolicyProvider
+    {
+        public override void ApplyPolicy(HttpContext httpContext)
+        {
+            if (Options.PermissionsPolicy.Length > 0)
+            {
+                httpContext.Response.Headers[SecurityHeaderNames.PermissionsPolicy] = String.Join(SecurityHeaderDefaults.PoliciesSeparator, Options.PermissionsPolicy);
```

Comment:
Idem, I think we should do some pre-processing to not have to do a `String.Join()` on maybe many strings, this on each request when the middleware call the `ApplyPolicy()` of providers.

Reply:
Sane as above